kroma-validator/challenge/status.go

```diff
@@ -1,8 +1,8 @@
 package challenge
 
 const (
-	// StatusNone is regarded as a challenge is not in progress.
-	// The other status are regarded as a challenge is in progress.
+	// StatusNone is regarded as a challenge that is not in progress.
+	// The other status is regarded as a challenge in progress.
```

Comment:
Other statuses mean that the challenge is in progress.
```suggestion
	// The others are regarded as a challenge in progress.
```